org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporterTest.java

```diff
@@ -487,6 +472,17 @@ public void testSubprojects() throws Exception {
 		}
 	}
 
+	@Test
+	@Ignore
```

Comment:
I'm trying to figure out why the buildship persistent file won't generated when running test cases...